webapp/packages/plugin-data-viewer/src/DatabaseDataModel/DataContext/DATA_CONTEXT_DV_ACTIONS.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { createDataContext } from '@cloudbeaver/core-data-context';
+
+import type { IDataTableActions } from '../../TableViewer/IDataTableActions.js';
+
+export const DATA_CONTEXT_DV_ACTIONS = createDataContext<IDataTableActions>('data-viewer-database-actions');
```

Comment:
should be in the view part

Reply:
also, you can split any actions that rely on view-specific context, so they will not be available without this context when other actions still be available